### PR TITLE
blog: September newsletter

### PR DESCRIPTION
**Pull request by GregSutcliffe**
Usual stuff, let me know what I've forgotten. Semi-AFK until Thursday (Rails Camp Scotland, yay!) but I'll try to pick up on feedback at least once before then.

### CODE REVIEW

File: _posts/2016-09-22-foreman-community-newsletter---september-2016.md [hunk 1]
```diff
@@ -0,0 +1,115 @@
...
+- newsletter
+- community
+excerpt: |
+  Foreman 1.13 highlights, Katello 3.1 and 3.2-RC1, EL6 deprecated for 1.14 and all the
+  usual plugin and media roundup
+---
+
+### Foreman 1.13 RC1 released
+
+Foreman 1.13 RC1 (and RC2 very soon!) is now available for general testing -
```
Comment (dLobatog) [inline _posts/2016-09-22-foreman-community-newsletter---september-2016.md]: RC2 is out already

File: _posts/2016-09-22-foreman-community-newsletter---september-2016.md [hunk 2]
```diff
@@ -0,0 +1,115 @@
...
+and this is a big one!  We've got the initial work on IPv6 (addressing and
+partial orchestration), UEFI bootloader support, improvements to DHCP subnet
```
Comment (dLobatog) [inline _posts/2016-09-22-foreman-community-newsletter---september-2016.md]: Grub2 worth mentioning here

File: _posts/2016-09-22-foreman-community-newsletter---september-2016.md [hunk 3]
```diff
@@ -0,0 +1,115 @@
...
+Another packed demo, great to see them building up again. This time we had some
+Roles/Orgs updates, Ansible & Remote Execution integration, Katello
+improvements and a new MS-DHCP provider for the smart proxy. Check it out!
+
+### Upcoming events
+
+#### [AnsibleFest Brooklyn - 11th October](https://www.ansible.com/ansiblefest)
+
+We're hoping to have a small presence at the next AnsibleFest in Brooklyn next
+month - if you're planning to go along, [get in touch][mail_greg] and if
```
Comment (dLobatog) [inline _posts/2016-09-22-foreman-community-newsletter---september-2016.md]: Maybe ask to ping @akofink here?

File: _posts/2016-09-22-foreman-community-newsletter---september-2016.md [hunk 4]
```diff
@@ -0,0 +1,115 @@
...
+If you love Config Management Camp in Ghent, but just can't wait until
+February, then you'll want to check out the one day event being organised in
+Berlin. Registration *and* the CFP are still open, so book in, and maybe even
+submit a Foreman talk of your own!
+
+_(All Foreman events are now available via the [Foreman Calendar](/events))_
+
+### Updated plugins
+
+- [foreman_bootdisk](https://github.com/theforeman/foreman_bootdisk) updated to 8.0.2
```
Comment (dLobatog) [inline _posts/2016-09-22-foreman-community-newsletter---september-2016.md]: Foreman Ansible had a pretty important update where running roles playbooks is now orchestrated by foreman-tasks, it's waiting on https://github.com/theforeman/foreman-packaging/pull/1335 and I'll make an announcement on the mailing list once it's merged

File: _posts/2016-09-22-foreman-community-newsletter---september-2016.md [hunk 5]
```diff
@@ -0,0 +1,115 @@
...
+
+The latest point release of the Foreman 1.12 series is now available, bringing
+bugfixes and support for Puppet Server 2.6. You can read all the details in the
+[announcement][1_12_3]
+
+[1_12_3]: https://groups.google.com/d/topic/foreman-users/0njkhkpRWzM/discussion
+
+### EL6 deprecated in 1.13, to be removed in 1.14
+
+Users of EL6 / Centos6 should be aware that 1.13 will be the last release with
```
Comment (dLobatog) [inline _posts/2016-09-22-foreman-community-newsletter---september-2016.md]: Add a disclaimer that RC packages are not available in the repos yet, people might look for them

**Comment by dLobatog** [comment]
:+1: @GregSutcliffe - check out the comments, will merge after that

File: _posts/2016-09-22-foreman-community-newsletter---september-2016.md (hunk 3 above)
Comment (GregSutcliffe) [reply _posts/2016-09-22-foreman-community-newsletter---september-2016.md]: @akofink you ok with that?

**Comment by GregSutcliffe** [comment]
@dLobatog updated, just pending an ACK from @akofink on whether he wants a namecheck :)

File: _posts/2016-09-22-foreman-community-newsletter---september-2016.md (hunk 3 above)
Comment (dLobatog) [reply _posts/2016-09-22-foreman-community-newsletter---september-2016.md]: I said him specifically because I know he's going btw

Comment (GregSutcliffe) [reply _posts/2016-09-22-foreman-community-newsletter---september-2016.md]: Indeed, I asked him to go :) - but not everyone would be comfortable with being singled out for contact in a public newsletter, so lets get permission first

Comment (akofink) [reply _posts/2016-09-22-foreman-community-newsletter---september-2016.md]: Yes, feel free to mention me!

**Comment by GregSutcliffe** [comment]
@dLobatog good to go!

**Comment by dLobatog** [comment]
Thanks @akofink @GregSutcliffe !

**Pull request merged**